doc/man1/openssl-configutl.pod.in

```diff
@@ -0,0 +1,63 @@
+=pod
+{- OpenSSL::safe::output_do_not_edit_headers(); -}
+
+=head1 NAME
+
+openssl-configutl - openssl configuration file helper command
+
+=head1 SYNOPSIS
+
+B<openssl> B<configutl>
+[B<-help>]
+[B<-config> I<filename>]
+[B<-out> I<filename>]
+[B<-noheader>]
+
+=head1 DESCRIPTION
+
+Real world openssl configuration files with support of B<.include> directive
+are quite complex, and there is a need to see the resulting system-wide configuration.
```

Comment:
This description tells us what the need is, but doesn't say what this utility actually does.